src/webvr.config.js

```diff
@@ -1,51 +0,0 @@
-export default {
```

Comment:
It wasn't possible to set because of the file we were using before (`webvr-polyfill/build/webvr-polyfill`)